app/Providers/LaravelTailwindConfigServiceProvider.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Providers;
+
+use ApprovedDigital\LaravelTailwindConfig\LaravelTailwindConfigServiceProvider as BaseProvider;
+
+class LaravelTailwindConfigServiceProvider extends BaseProvider
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function register()
+    {
+        $this->registerTailwindService();
+
+        // @todo if Acorn registers globals before register providers we can
+        // remove this overload
```

Comment:
At the moment https://github.com/roots/acorn isn't entirely compatible with Laravel packages as global functions such as `base_path()` aren't available until after providers have loaded. This requires us to swap out the provider for our own and remove the code that uses these functions. We could also have used `Roots\` namespaced functions here but the functionality lost isn't so important. I brought it up on Roots slack channel and they _might_ be working on a fix, in which case we can remove this provider and use the upstream one directly.